common/src/main/java/org/commonjava/maven/ext/common/util/LineSeparator.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2012 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.commonjava.maven.ext.common.util;
+
+/**
+ * Partially taken from org.jdom2.output.LineSeparator as the Maven Release Plugin uses JDOM1 not 2.
+ */
```

Comment:
Should the file have the original copyright/license header then?

Reply:
Ok